pkg/datastore/documentloader/pdf.go

```diff
@@ -4,14 +4,20 @@ import (
 	"context"
 	"io"
 	"strings"
+	"sync"
 
 	md "github.com/JohannesKaufmann/html-to-markdown"
 	"github.com/PuerkitoBio/goquery"
 	"github.com/gen2brain/go-fitz"
 	"github.com/gptscript-ai/knowledge/pkg/datastore/types"
+	"github.com/gptscript-ai/knowledge/pkg/env"
 	vs "github.com/gptscript-ai/knowledge/pkg/vectorstore"
+	"golang.org/x/sync/errgroup"
 )
 
+// PDF_LOAD_PARALLEL_THREAD can be set as an environment variable that controls the number of goroutines to load pdf pages
+const PDF_LOAD_PARALLEL_THREAD = "PDF_LOAD_PARALLEL_THREAD"
```

Comment:
Maybe something more "reusable" like `DOCUMENTLOADER_<Loader Name>_<Setting>` ?

E.g. `DOCUMENTLOADER_PDF_NUM_THREADS` ?

Reply:
Or maybe even as a PDF option so one can set it via config?

Reply:
yeah, I think putting into pdf option is a good way. Will address 👍